src/vs/workbench/parts/search/browser/searchViewlet.ts

```diff
@@ -175,6 +175,7 @@ export class SearchViewlet extends Viewlet {
 		const exclusionsUsePattern = this.viewletSettings['query.exclusionsUsePattern'];
 		const includesUsePattern = this.viewletSettings['query.includesUsePattern'];
 		const patternIncludes = this.viewletSettings['query.folderIncludes'] || '';
+		const userExtended = this.viewletSettings['query.userExtended'] || '';
```

Comment:
Looks great, only comment is that the name `userExtended` isn't totally obvious at first. Maybe `queryDetailsExpanded` or something like that?